docs/sources/panels/transformations/apply.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Apply a Transformation"
```

Comment:
```suggestion
title = "Apply a transformation"
```

Reply:
All titles and headings should be in sentence case.

Reply:
Fixed